bin/test.sh

```diff
@@ -58,7 +58,17 @@ function test {
     cp "${exercise_dir}/.meta/${example_file}" "${outdir}/${exercise_file}.${file_ext}"
   fi
 
-  eval "${PHPUNIT_BIN}" --no-configuration "${outdir}/${test_file}"
+  # The time-limit `54` is based on an approximation of the performance.
+  # Online Test Runner is appr. 3x faster than GitHub CI on Ubuntu.
+  #
+  # The total runtime limit of the Online Test Runner is 20 seconds including
+  # Docker container launch and processing the results. That leaves appr. 18
+  # seconds for all tests of an exercise.
+  #
+  # Putting that together gives a max. of 18 seconds x3 = 54 seconds for any
+  # test class in CI.
+  # See: https://forum.exercism.org/t/test-tracks-for-the-20-seconds-limit-on-test-runners/10536/8
```

Comment:
I think "Exercism Test Runner" would be more precise than "Online Test Runner", I also suggest the following comment, I think it is clearer:

```suggestion
  # `54s` timeout is an approximation to ensure the tests will not timeont in Exercism Test Runner.
  #
  # 1. Exercism Test Runner is around 3 times faster than GitHub CI on Ubuntu.
  #    See: https://forum.exercism.org/t/test-tracks-for-the-20-seconds-limit-on-test-runners/10536/8
  # 2. Exercism Test Runner should complete in 20s and involves:
  #    - Starting Docker container (~1s)
  #    - Running the test suite
  #    - Processing the results (~1s)
  #
  # This gives 18s maximum for the test suite to run in the Exercism Test Runner.
  # Hence the test suite should complete in less than 18s x 3 = 54s in GitHub CI on Ubuntu.
```

The current comment still is okay, so feel free to update it or no (or even suggest another version).